src/services/rocketChat/index.js

```diff
@@ -352,8 +353,8 @@ class RocketChatChannel {
 		return this.app.service('teams').get(teamId, internalParams)
 			.then((team) => {
 				currentTeam = team;
-				const userNamePromises = currentTeam.userIds.map(user => this.app.service('rocketChat/user').get(user.userId));
-
+				const userNamePromises = currentTeam.userIds.map(user => this.app.service('rocketChat/user').get(user.userId)
+					.catch(Promise.resolve));
```

Comment:
not required, because the catch is within the map function.
Even if one or more users fail, the map still returns an array that reaches 359, and execution continues.